src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-dropdown-property/elsa-dropdown-property.tsx

```diff
@@ -48,7 +48,7 @@ export class ElsaDropdownProperty {
     const fieldId = propertyName;
     const fieldName = propertyName;
     let currentValue = this.currentValue;
-    const {items} = this.selectList;
+    const { items } = this.selectList.items;
```

Comment:
This should not work - if it does, it means the contents of the `this.selectList` contains an unexpected value (an object containing `items` which itself then also has an `items` field.

I just pushed an update that seems to fix the runtime select list items story (I tried it with a sample custom activity with runtime items implemented).

Please give it a go and let me know if it works or not.

Thanks!

Reply:
ok - just did a compare with other implementations - and this was the only where this.selectlist was used - the other editor are using selectlist.items.

Reply:
and btw - the webservice call worked - i got a javascript exception - but i try again once the build is finished

![image](https://user-images.githubusercontent.com/158478/143878692-5da8772e-dfbf-44c7-a68c-3c83893f9e98.png)

Reply:
works perfect now - sorry for my wrong assumption :)